pp.go

```diff
@@ -69,6 +73,34 @@ func Fatalln(a ...interface{}) {
 	os.Exit(1)
 }
 
+/*
+
+Change Print* functions output to o
+For example, you can limit output by ENV followings
+
+	func init() {
+		if os.Getenv("DEBUG") == "" {
+			pp.SetDefaultOutput(os.DevNull)
```

Comment:
By the way, `os.DevNull` can't be used as `io.Writer` because it is a string ("/dev/null").
I've just known that [ioutil package](http://golang.org/pkg/io/ioutil/#pkg-variables) has `Discard` variable and it may be proper as output suppression. https://github.com/k0kubun/pp/commit/009142fd9edb90a99058b8667acd15ad3ed4cccd